### DISCUSSION TITLE
Passkey FIDO2/Webauthn Login

### DISCUSSION THREAD

**Discussion by steverding** [nilsteampassnet/TeamPass#4182] [Ideas]
There was an old Yubikey login option that seems to be gone, but what really would be cool is a FIDO2/Passkey login option and use the security key feature of the yubikey.

**Comment by reginfo** [comment]
Hi, 

Just wanted to UP this question about Yubikey. Is it in roadmap?

**Comment by guerricv** [comment]
Hello,
I will have a look on that soon, I got one, I can give it a try.
Physical security devices are in the mood currently. It could be a good thing to recall that function in TeamPass.
